src/tests.rs

```diff
@@ -1,3 +1,4 @@
+#![deny(missing_docs)]
```

Comment:
There shouldn't be a `#![deny(missing_docs)]` in test files.

Reply:
woops, clearly i tried to do this way too fast